exec/java-exec/pom.xml

```diff
@@ -782,8 +782,7 @@
       <id>linux-amd64</id>
       <activation>
         <os>
-          <arch>amd64</arch>
-          <name>Linux</name>
+          <arch>x86_64</arch>
```

Comment:
It is only macOS that returns x86_64, IIRC.  Windows and Linux return amd64 for this arch.  I think the following will be enough to fix the Travis problem...
```
diff --git a/exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java b/exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java
index 35335c2db..0ff029424 100644
--- a/exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java
+++ b/exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java
@@ -1001,8 +1001,7 @@ public class TestParquetWriter extends ClusterTest {
   // Only attempt this test on Linux / amd64 because com.rdblue.brotli-codec
   // only bundles natives for Mac and Linux on AMD64.  See PARQUET-1975.
   @Test
-  @EnabledIfSystemProperty(named = "os.name", matches = "Linux")
-  @EnabledIfSystemProperty(named = "os.arch", matches = "amd64")
+  @Ignore("Depends on platform specific library code.")
   public void testTPCHReadWriteBrotli() throws Exception {
     try {
       client.alterSession(ExecConstants.PARQUET_WRITER_COMPRESSION_TYPE, "brotli");
```